clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -545,6 +545,9 @@ getTripleBasedSYCLPostLinkOpts(const ArgList &Args,
   else
     PostLinkArgs.push_back("-spec-const=emulation");
 
+  if (Triple.isSPIROrSPIRV() || SYCLNativeCPU)
```

Comment:
Should we check if -ir-output-only is passed in via -sycl-post-link-options?

Reply:
Thanks for the review. Hopefully I'm understanding the comment correctly: Neither `-properties` nor `-ir-output-only` will ever be part of `-sycl-post-link-options` because both are added in `getTripleBasedSYCLPostLinkOpts` inside the driver, which is not called to compute `-sycl-post-link-options` when constructing the `clang-linker-wrapper` job. The only way it could end up there is from `-Xdevlce-post-link` from the user, and we don't seem to be checking for that case elsewhere in this function so I'm not sure we should do it only for those arguments.

Reply:
Thanks for the clarification @sarnex. I verified it.